### ISSUE TITLE
Standalone zero-to-working-pmtiles example

### ISSUE THREAD

**Issue by mikaelhg** [protomaps/PMTiles#71]
Here's the most basic HOWTO possible for getting from zero to a working protomaps web application.

Easy to adapt for use with any openmaptiles JSON style, just replace the layers with your layers.

https://gist.github.com/mikaelhg/edf65dfad43e240fb4c483587b5e7f93

**Comment by sntran** [comment]
Thanks for the step-by-step guide. 

Considering that "Our real-world, planet-scale dataset can address over 350 million individual tiles in just 91.6 megabytes", it would be nice to have a download of such "planet.pmtiles" ready-made.

**Comment by mikaelhg** [comment]
I don't think pmtiles 2.1.0 can yet produce files in the v3 format to which the article you quoted refers. Certainly the finland.pmtiles file I produced is around 700 MB if memory serves.

**Comment by bdon** [comment]
@mikaelhg you're right, the implementation of v3 is still WIP as I need to create both the writer tooling and the reader libraries. Will do another pass at updated docs/examples when I get a chance. (also moving this to a discussion!)
